client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -167,7 +168,7 @@ const LoggedOutForm = React.createClass( {
 		return (
 			<LoggedOutFormLinks>
 				<LoggedOutFormLinkItem href={ login( { redirectTo } ) }>
-					{ this.translate( 'Already have an account? Sign in' ) }
+					{ i18n.translate( 'Already have an account? Sign in' ) }
```

Comment:
@aduth In this case I believe I could make it work without the linter requiring me to rewrite to an es6 class. However, there are several cases in this PR where that won't be possible, which is why I prefer to abandon this large PR and work on a bunch of smaller ones. Making a very large PR is exactly what I'm trying to avoid here.